packages/manager/modules/pci/src/projects/projects.routing.js

```diff
@@ -123,19 +123,23 @@ export default /* @ngInject */ ($stateProvider) => {
         }),
 
       goToState: /* @ngInject */ ($state) => (targetedStatePromise) => {
-        targetedStatePromise.then((targetedState) => {
-          if (targetedState.isUApp) {
-            const url = new URL(targetedState.url);
-            Object.keys(targetedState.params).forEach((key) => {
-              url.searchParams.append(key, targetedState.params[key]);
-            });
-            const state = [targetedState.url, url.searchParams].join('?');
-            window.location = state;
-            return state;
-          }
-          const { state, params, options } = targetedState;
-          return $state.go(state, params, options);
-        });
+        if (targetedStatePromise instanceof Promise) {
```

Comment:
**question:** Can't we call the state navigation with promise after project creation?